nav2_smac_planner/src/smac_planner_lattice.cpp

```diff
@@ -282,11 +298,27 @@ nav_msgs::msg::Path SmacPlannerLattice::createPlan(
   NodeLattice::CoordinateVector path;
   int num_iterations = 0;
   std::string error;
-
+  std::unique_ptr<std::vector<std::tuple<float, float, float>>> expansions = nullptr;
+  if (_debug_visualizations) {
+    expansions = std::make_unique<std::vector<std::tuple<float, float, float>>>();
+  }
   // Note: All exceptions thrown are handled by the planner server and returned to the action
   if (!_a_star->createPath(
       path, num_iterations, _tolerance / static_cast<float>(_costmap->getResolution())))
   {
+    if (_debug_visualizations) {
+      geometry_msgs::msg::PoseArray msg;
```

Comment:
These 2 blocks are not covered by any unit or system tests, we need to have that in order to merge the PR for quality control requirements

Reply:
The WPF test failed oddly, so I retriggered CI to see if that was just a fluke